src/components/CourseCard/CourseCard.module.css

```diff
@@ -0,0 +1,19 @@
+.root {
+  max-width: 280px;
```

Comment:
Reminder to remove max-width when using it in grid system

Reply:
ill add a todo for the person who work on this later